templates/projects/empty/project.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.0.0-*",
 
   "dependencies": {
-    "Microsoft.AspNet.Server.IIS": "1.0.0-beta5",
-    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta5",
-    "Kestrel": "1.0.0-beta5"
+    "Microsoft.AspNet.Server.IIS": "1.0.0-beta6",
+    "Microsoft.AspNet.Server.WebListener": "1.0.0-beta6",
+    "Kestrel": "1.0.0-*"
```

Comment:
I suppose so.  I was basing it on the template files in the Zip provided.  Is there an inconsistency in the template files that were provided?  I noticed the Web App Simple (Basic) project.json is similar (not specifying the beta6 tag).